libs/cri-storing-service/src/main/java/uk/gov/di/ipv/core/library/cristoringservice/CriStoringService.java

```diff
@@ -79,6 +79,7 @@ public void recordCriResponse(
                 journeyRequest.getIpvSessionId(),
                 journeyRequest.getIpAddress(),
                 journeyRequest.getDeviceInformation(),
+                null,
```

Comment:
Can we use `journeyRequest.getFeatureSet()`? (You might need to use `RequestHelper.getFeatureSet` to parse it)

Reply:
Done